### DISCUSSION TITLE
Support Nitro Module

### DISCUSSION THREAD

**Discussion by baronha** [callstack/react-native-builder-bob#650] [Ideas]
Marc just released the [Nitro Module](https://nitro.margelo.com/docs/what-is-nitro) and I found it suitable for implementation. What do you think about this?

**Comment by atlj** [comment]
Nitro Modules are supported now 🎉 https://github.com/callstack/react-native-builder-bob/pull/721

**Comment by baronha** [comment]
Let's GOOOOOO. A good news today !!!! Superrrrr
